samples/IISSample/Startup.cs

```diff
@@ -12,6 +13,7 @@ public class Startup
     {
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddAuthentication();
```

Comment:
Didn't try running the samples, perhaps mistakenly I assumed functional tests were a superset, urgh yeah forgot we can't depend on security in this repo :/